packages/cli/src/CommandsLogic.ts

```diff
@@ -585,7 +586,7 @@ export class CommandsLogic {
     if (deployOptions.deployPaymaster ?? false) {
       pmInstance = await this.deployPaymaster({ ...options }, rInstance.options.address, fInstance, deployOptions.skipConfirmation)
     }
-    await registerForwarderForGsn(fInstance, options)
+    await registerForwarderForGsn(defaultGsnConfig.domainSeparatorName, fInstance, options)
```

Comment:
Hm... if any project can register whatever "domain separator" and no contract code is there to validate it, then maybe we should change the forwarder to ignore the "domain separator" altogether....

Reply:
never mind.
GSN doesn't require the domain-separator to be valid, since the request in an elaborate RelayRequest, which is good enough.